packages/website/content/integrations/examples/RegisterIntegration.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import {
+  Checkbox,
+  Form,
+  FormControl,
+  Radio,
+  Select,
+  Textarea,
+  TextInput,
+} from '@contentful/f36-forms';
+import { Button } from '@contentful/f36-components';
+
+export default function ReactHookForm() {
```

Comment:
💅 Could you add please `Example` to the file name as well, just for the sake of consistency.
```suggestion
export default function RegisterIntegrationExample() {
```